Commons Box Minecraft/src/main/java/pl/plajerlair/commonsbox/minecraft/compat/events/Events.java

```diff
@@ -50,7 +53,14 @@ public void onEntityPickupItem(EntityPickupItemEvent event) {
 
   @EventHandler
   public void onPlayerPickupArrow(PlayerPickupArrowEvent event) {
-    CBPlayerPickupArrow cbEvent = new CBPlayerPickupArrow(event.getPlayer(), event.getItem(), (Projectile) event.getArrow(), event.getRemaining(), VersionUtils.isPaper() ? event.getFlyAtPlayer() : false);
+    Projectile projectile;
+    try {
+      projectile = (Projectile) event.getClass().getDeclaredMethod("getArrow").invoke(event);
+    } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
+      e.printStackTrace();
+      return;
```

Comment:
Instead of throwing exception and returning, we should call `CBPlayerPickupArrow` with `false` parameter if it fails.